mamba/mamba.py

```diff
@@ -594,28 +598,29 @@ def install(args, parser, command="install"):
             exit_code = 1
             return exit_code
 
-        package_cache = api.MultiPackageCache(context.pkgs_dirs)
-        transaction = api.Transaction(
-            solver, package_cache, PackageCacheData.first_writable().pkgs_dir
-        )
-        mmb_specs, to_link, to_unlink = transaction.to_conda()
+        with FileLock(PackageCacheData.first_writable().pkgs_dir):
+            package_cache = api.MultiPackageCache(context.pkgs_dirs)
+            transaction = api.Transaction(
+                solver, package_cache, PackageCacheData.first_writable().pkgs_dir
+            )
+            mmb_specs, to_link, to_unlink = transaction.to_conda()
 
-        specs_to_add = [MatchSpec(m) for m in mmb_specs[0]]
-        specs_to_remove = [MatchSpec(m) for m in mmb_specs[1]]
+            specs_to_add = [MatchSpec(m) for m in mmb_specs[0]]
+            specs_to_remove = [MatchSpec(m) for m in mmb_specs[1]]
```

Comment:
I guess the tabs are causing CI error on formatting

Reply:
maybe also the imports, see https://github.com/mamba-org/mamba/pull/1105/checks?check_run_id=3201260151

Reply:
I can't reproduce the formatting issues locally, only the isort one, which should be fixed in this last commit.